homeassistant/components/http/__init__.py

```diff
@@ -116,7 +116,21 @@
     is_ban_enabled = conf[CONF_IP_BAN_ENABLED]
     login_threshold = conf[CONF_LOGIN_ATTEMPTS_THRESHOLD]
 
-    if api_password is not None:
+    if api_password == 'welcome':
+        _LOGGER.warning('Using default password, this is not recommended!')
+        hass.components.persistent_notification.async_create(
```

Comment:
We should not make this a persistent notification. That is super annoying if this is the behavior that you want.

Reply:
@balloob It's not clear to me what this entails, I see a message in overview which I can dismiss until the next restart of hass, which seemed not to intrusive to me (but I'll add a config option to overwrite it, somethink like: `no_password_i_know_what_i_am_doing: true`

But if you have a better proposition of a type of notification to show, feel free to inform me, I'm new to home assistant and haven't even really used it, just wanted to make sure nobdy accidentally started it without password and was unaware, like I just did the first time I started it a few days ago.

Reply:
I don't want extra config options either. Just remove the persistent notification.

We're discussing [extra HTTP auth options](https://github.com/home-assistant/architecture/issues/17) and will also introduce an onboarding flow for people to set up auth so this will be removed in the future, hence I don't want to do a breaking change just to remove a config option.